sbncode/LArG4/MergeSimSourcesSBN_module.cc

```diff
@@ -300,11 +322,12 @@ void sbn::MergeSimSourcesSBN::produce(art::Event& e)
 
   for (auto const& [i_source, input_label] : util::enumerate(fInputSourcesLabels)) {
 
+    auto const input_partCol = e.getValidHandle<std::vector<simb::MCParticle>>(input_label);
+    art::PtrMaker<simb::MCParticle> const makePartPtr{e};
```

Comment:
It seems that this object is used only in the `if (fFillMCParticlesAssociated)` branch of the code, so I suggest it is moved into that branch.

Reply:
done! thanks for the comment i just didn't see it until now..